crates/turbopack-node/src/evaluate.rs

```diff
@@ -140,13 +140,28 @@ pub async fn evaluate(
         debug,
     )
     .await?;
-    let mut operation = pool.operation().await?;
+
     let args = args.into_iter().try_join().await?;
-    operation
-        .send(EvalJavaScriptOutgoingMessage::Evaluate {
-            args: args.iter().map(|v| &**v).collect(),
-        })
-        .await?;
+
+    // Sending to workers can fail if reused workers are in a bad state.
+    // To we retry picking workers from the pools until we succeed.
+    let mut i = 1;
+    let mut operation = loop {
+        let mut operation = pool.operation().await?;
```

Comment:
`pool.operation()` should not return a process in a "bad state"

Reply:
Not sure how to ensure that. We could check the worker with some ping message, but that costs a roundtrip and it still could crash right after that, so we would still need to handle this case and retry.

Reply:
At the very least, we could check `child.try_wait()`